### ISSUE TITLE
x64 shell integration points to x86 exe

### ISSUE THREAD

**Issue by ck341** [WinMerge/winmerge#1397]
Just installed version 2.16.20 x64, and the shell integration did not work. I noticed that the registry value **HKEY_CURRENT_USER\SOFTWARE\Thingamahoochie\WinMerge\Executable** was set to **C:\Program Files (x86)\WinMerge\WinMergeU.exe**, which did not exist on my system. Manually changing it to **C:\Program Files\WinMerge\WinMergeU.exe** fixed the integration without requiring a reboot.

**Comment by sdottaka** [comment]
The `HKEY_CURRENT_USER\SOFTWARE\Thingamahoochie\WinMerge\Executable` is rewritten in the following cases
- When installed by the installer
- When uninstalling (registry keys are deleted)
- When you press the OK button in the WinMerge Options dialog.

After installing the 64-bit version, did you install the 32-bit version or start the 32-bit version of WinMerge?

**Comment by ck341** [comment]
No, I didn't install or use WinMerge x86 after installing the x64 version. I forgot to mention that I uninstalled WinMerge before installing it, and I forget which version was uninstalled, but I currently have a Program Files (x86)\WinMerge folder with a single file ShellExtension64.dll.

**Comment by sdottaka** [comment]
I have not been able to reproduce this issue. If you know the procedure to reproduce this problem, please let me know.
